cirrus/server/cirrus/main.py

```diff
@@ -109,6 +110,39 @@ def initialize_glean():
     return pings, metrics
 
 
+def get_metrics_data(
+    enrolled_partial_configuration: Dict[str, Any]
+) -> List[tuple[str, str, str]]:
+    events: List[Dict[str, Any]] = enrolled_partial_configuration.get("events", [])
+    data: List[tuple[str, str, str]] = []
+    for event in events:
+        if event.get("change") == "Enrollment":
+            experiment_slug: str = event.get("experiment_slug", "")
+            branch_slug: str = event.get("branch_slug", "")
+            experiment_type: str = app.state.remote_setting.get_recipe_type(
```

Comment:
Does the type checker yell if you omit these annotations? They should be able to be inferred.

Reply:
Yep it's all good now